<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1499</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>700</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>190</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>1441</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите граф:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>1301</width>
     <height>159</height>
    </rect>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>122</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>31</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>35</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="0" column="6">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="0" column="7">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="0" column="8">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="0" column="9">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="0" column="10">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="0" column="11">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="0" column="12">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="0" column="13">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="0" column="14">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="0" column="15">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="0" column="16">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="0" column="17">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="0" column="18">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="0" column="19">
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item row="0" column="20">
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item row="0" column="21">
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="1" column="4">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="1" column="5">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="1" column="6">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="1" column="7">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="1" column="8">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="1" column="9">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="1" column="10">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="1" column="11">
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item row="1" column="12">
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item row="1" column="13">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="1" column="14">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="1" column="15">
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item row="1" column="16">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="1" column="17">
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item row="1" column="18">
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item row="1" column="19">
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item row="1" column="20">
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item row="1" column="21">
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="2" column="4">
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item row="2" column="5">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="2" column="6">
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item row="2" column="7">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="2" column="8">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="2" column="9">
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item row="2" column="10">
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item row="2" column="11">
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item row="2" column="12">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="2" column="13">
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item row="2" column="14">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="2" column="15">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="2" column="16">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="2" column="17">
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item row="2" column="18">
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item row="2" column="19">
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item row="2" column="20">
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item row="2" column="21">
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>200</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Максимальный поток:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>330</y>
     <width>700</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>194</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Максимальный поток</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>165</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Алгоритм Флойда</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>133</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Исходный граф:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>290</y>
     <width>174</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Получившийся граф:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>259</y>
     <width>1441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>330</y>
     <width>691</width>
     <height>401</height>
    </rect>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>11</number>
   </property>
   <property name="columnCount">
    <number>11</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>33</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <zorder>label</zorder>
  <zorder>frame</zorder>
  <zorder>label_3</zorder>
  <zorder>tableWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_4</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>line</zorder>
  <zorder>pushButton</zorder>
  <zorder>tableWidget_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
